web/templates/caseload/index.html.eex

```diff
@@ -0,0 +1,17 @@
+<h2 class="tc">Caseload</h3>
+
+<%= for user <- @hl_users do %>
+  <div class="w-100 hl-bg-grey pa3 mv2">
+    <p class="b di"><%= user.name %></p><span class="fr">></span>
+  </div>
+<% end %>
+
+<%= for user <- @non_hl do %>
+  <div class="w-100 ph3 pv2">
+    <p>
+       <p class="b di"><%= user."Forename" %> <%= user."Surname" %></p>
```

Comment:
Hmmm we might be able to do some metaprogramming to create helper functions with more standard naming. So we could just call `user.forename`, but underneath that would call the above. Might make this a bit tidier?